### DISCUSSION TITLE
Multiple commit --filter don't work

### DISCUSSION THREAD

**Discussion by wanschi** [vercel/turborepo#3173] [Help] [answered]
Hello,

we would like to integrate following behavior to our github actions ci/cd pipeline:

Whenever a PR is opened a script should run. The constraints should be: Run only when the app package is affected (--filter=app) and the app differs from the main branch (--filter=[origin/main]) and also the app package was modified in comparision to the previous commit: (--filter=[HEAD^1]). The last should make the the script only once when the PR was opened instead of running it every commit after feedback from a code review was implemented.

This is the full command which is not successful for us: 
`turbo run deploy-native-dev-client --filter=app --filter=[origin/main] --filter=[HEAD^1]`

Thank you in advance!

**Comment by mehulkar** [comment, marked as answer]
Hey @DanielJanke thanks for reporting this. Could you file an issue instead of a discussion with a reproduction repo?

**Reply by mehulkar** [reply]
cc @wanschi fyi looks like your username changed
